### ISSUE TITLE
Investigate top/bottom positions in drawLotto

### ISSUE THREAD

**Issue by shippy** [YaleDecisionNeuro/PsychTaskFramework#76]
Something screwy is going on with the top/bottom designations in `lib/draw/drawLotto.m` (and, in general, `tasks/XYZ/XYZ_drawTask.m` -- specifically, I'm not sure that `top` is always on top? This doesn't have any consequences for data collection, but I should still spend some time looking into this, if only so that I catch the gotcha before future alterations.

**Comment by shippy** [comment]
I take that back: this does have consequences for data collection and needs to be investigated right away. Time to roll out `git bisect` to find out when this started happening.

**Comment by shippy** [comment]
Okay. UVRA was always recording the winning probability incorrectly; SODM (and, presumably, all other tasks) have been broken in 1fd72dcce0902367f16b954f4e10ecf8b2de0da1 on March 11. The fix should be easy, but let's be thorough and get down to the correctness of color, too.

**Comment by shippy** [comment]
The plot thickens: RA and UVRA have been wrong all along, which is likely the source of the issue in `drawLotto`. SODM and MDM only broke in 1fd72dc. Here's the relevant difference that they have one commit before that, in bcc99c9adaf308b39c04bec95085afb8eafa59fb:

```matlab
% MDM_drawTask.m
topTxtCoords = [W/2 - txtDims1(1)/2, Y1 - txtDims1(2)/2];
bottomTxtCoords = [W/2 - txtDims2(1)/2, Y3 + txtDims2(2)];
DrawFormattedText(windowPtr, txt1, ...
  topTxtCoords(1), topTxtCoords(2), blockSettings.objects.lottery.stakes.fontColor);
DrawFormattedText(windowPtr, txt2, ...
  bottomTxtCoords(1), bottomTxtCoords(2), blockSettings.objects.lottery.stakes.fontColor);
% -> y: top @ y1 - txtDims1(2)/2, bottom @ y3 + txtDims2(2)/2
``` 

```matlab
% RA_drawTask.m
xCoords = [W/2 - digitWidth(1)/2, W/2 - digitWidth(2)/2];
yCoords = [Y3, Y1 - digitHeight(1)] + randomConstantAdjustment;
DrawFormattedText(windowPtr, dollarFormatter(amtOrder(1)), ...
  xCoords(1), yCoords(1), blockSettings.objects.lottery.stakes.fontColor);
DrawFormattedText(windowPtr, dollarFormatter(amtOrder(2)), ...
  xCoords(2), yCoords(2), blockSettings.objects.lottery.stakes.fontColor);
% -> y: top @ y3, bottom @ y1 - digitHeight
```

I'm reasonably sure that the difference isn't coming from the derivation of payoff value, although that should be checked.

(Possibility: the color might be wrong, too. This might not be that much of a big deal. It is worth noting that in the latest master, the color order and the color description order is inverted, so it's unclear what the actual color association ought to be.)

**Comment by shippy** [comment]
Closing, as the question is resolved, and continuing work in #115.
